ydb/core/protos/counters_hive.proto

```diff
@@ -44,6 +45,7 @@ enum ECumulativeCounters {
     COUNTER_SUGGESTED_SCALE_UP = 10 [(CounterOpts) = {Name: "SuggestedScaleUp"}];
     COUNTER_SUGGESTED_SCALE_DOWN = 11 [(CounterOpts) = {Name: "SuggestedScaleDown"}];
     COUNTER_STORAGE_BALANCER_EXECUTED = 12 [(CounterOpts) = {Name: "StorageBalancerExecuted"}];
+    COUNTER_TABLETS_REASSIGNED = 13 [(CounterOpts) = {Name: "TabletsReassigned"}];
```

Comment:
`TabletsReassigned` quite possibly will be confused with `TabletsMoved`.
May be there is a more accurate name for tablet-storage groups assignment?

Reply:
Added the word 'storage' to counter name.